pkg/controller/machinehealthcheck/machinehealthcheck_controller.go

```diff
@@ -706,6 +722,35 @@ func (t *target) remediationStrategyExternal(r *ReconcileMachineHealthCheck) err
 	return nil
 }
 
+func (t *target) removeRemediationStrategyExternal(r *ReconcileMachineHealthCheck) error {
+	// external annotation does not exist on the machine, nothing to do
+	if _, ok := t.Machine.Annotations[machineExternalAnnotationKey]; !ok {
```

Comment:
(Nice to have) - Might be worth checking `t.Machine.Annotations` isn't nil as a safeguard

Reply:
will do 👍🏼